cpp/thirdparty/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ endif()
 
 # ## BOOST
 if(MEMILIO_USE_BUNDLED_BOOST)
-    message(STATUS "Downloading Boost library. This may take some time.")
+    message(STATUS "Downloading Boost library. This may take several minutes. This is the time to get yourself a coffee. :-)")

```

Comment:
```suggestion
    message(STATUS "Downloading Boost library. This may take several minutes. This is the time to get yourself a tea. :-)")
```
Fixed a typo ;)